### PR TITLE
Improve processInactivityUpdates

### PR DESCRIPTION
**Pull request by twoeths**
**Motivation**

+ `processInactivityUpdates` is slow and almost equal to `processRewardsAndPenalties` even we don't update anything
+ the profile is shown in #3173

**Description**

1. Loop through the persistent-ts array
2. New method `setMultiple` so that we go through the proxy only 1 time. We'll enhance `setMultiple` in another PR to build the whole tree from scratch if we have to update so many values.
3. Avoid importing functions/constants from other modules inside for loop

This gives 5x-7x improvement when I run altair and 3x in the performance test. Note that the improvement #3 gives only 10% speed increasement but we can apply same strategy for other functions 

resolves issue 1 of #3173

### CODE REVIEW

**Comment by twoeths** [comment]
```
pyrmont_e62330 - altair processInactivityUpdates | 8.9175 ms/op | 72.662 ms/op | 0.12
```

@dapplion looks like we miss the nice performance improvement alert in the old version of benchmark tool

**Comment by dapplion** [comment]
> ```
> pyrmont_e62330 - altair processInactivityUpdates | 8.9175 ms/op | 72.662 ms/op | 0.12
> ```
> 
> @dapplion looks like we miss the nice performance improvement alert in the old version of benchmark tool

It will only show up if the improvement is better than the current threshold. In CI that's 3, so the factor in CI has to be lower than 0.33.

Also the benchmark `pyrmont_e62330 - altair processInactivityUpdates` has a threshold of Infinity, so it won't ever show here as an improvement

File: packages/beacon-state-transition/test/perf/altair/epoch/processInactivityUpdates.test.ts [hunk 1]
```diff
@@ -21,6 +21,10 @@ import {mutateInactivityScores} from "./util";
 
 describe("altair processInactivityUpdates", () => {
   const vc = numValidators;
+  setBenchOpts({
+    maxMs: 60 * 1000,
+    minMs: 50 * 1000,
```
Comment (dapplion) [inline packages/beacon-state-transition/test/perf/altair/epoch/processInactivityUpdates.test.ts]: Why are this parameters necessary?

Comment (twoeths) [reply packages/beacon-state-transition/test/perf/altair/epoch/processInactivityUpdates.test.ts]: I read the benchmark result wrong, thought the test was stable.
Reverted the change.

**Review by dapplion — APPROVED** [review]

**Pull request merged**